frontEnd/src/firebase.js

```diff
@@ -0,0 +1,24 @@
+import firebase from "firebase/app";
+//const env = process.env;
+
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+//ideally this would be hidden behind a gitignore but need to get .env file working, there is a value parsing error (known error for firebase). Not sure how to resolve it at this time.
+//source : https://github.com/mortezasabihi/vue-firebase-auth/blob/master/src/firebase.js 
+const firebaseConfig = {
+  apiKey: "AIzaSyCKv8JlZisXPsuu3RMIpY3o13OCZ8h65A8",
+  authDomain: "bbbs-applicant-portal.firebaseapp.com",
+  databaseURL: "https://bbbs-applicant-portal.firebaseio.com",
+  projectId: "bbbs-applicant-portal",
+  storageBucket: "bbbs-applicant-portal.appspot.com",
+  messagingSenderId: "4468871993",
+  appId: "1:4468871993:web:97201aa3c2304e07080ee8",
+  measurementId: "G-PERE0FG5PD"
+};
```

Comment:
This really should not be committed to the repository cause this is a security breach. Now our authentication API keys are open to the public. We will have to delete this key and create a new one that is not committed.  Why not just use a config file like we do for the backend?

Reply:
yes good point, that is really my bad I have removed cached and added to gitignore, we should move it to a new firebase project at the end for sure. Also I tried to use the config file like Ben did but it wasn't connecting properly (I was probably doing something wrong idk) so I got it working.